smallvil/src/winit.rs

```diff
@@ -24,10 +22,15 @@ pub fn init_winit(
     let display_handle = &mut data.display_handle;
     let state = &mut data.state;
 
-    let (mut backend, mut winit) = winit::init()?;
+    let (mut backend, winit_source) = winit::WinitGraphicsBackend::new(GlAttributes {
+        version: (3, 0),
```

Comment:
Off topic for this pull request, this seems to have always been 3.0, but the winit backend should be possible even with gles 2.0?

@Drakulix?